generators/liquibase/support/prepare-field.js

```diff
@@ -139,14 +139,27 @@ function parseLiquibaseLoadColumnType(entity, field) {
 }
 
 export default function prepareField(entity, field) {
+  field.fieldDefaultValueDefined = field.defaultValue !== undefined;
+  field.fieldHasAnyDefaultValue = field.fieldDefaultValueDefined || field.defaultValueComputed !== undefined;
+
```

Comment:
Every use of `fieldDefaultValueDefined` and `fieldHasAnyDefaultValue` happens inside liquibase generator and can be replaced with `liquibaseDefaultValueAttributeValue` now.

Reply:
Done.